### DISCUSSION TITLE
Getting provider from wagmi for web3js

### DISCUSSION THREAD

**Discussion by 2facedsquid** [wevm/wagmi#1853] [Q&A]
Hello I have recently switched to the walletConnect V2 React + wagmi SDK beforehand I was using only metamask so the provider was extracted from the window.ethereum when creating a new Web3 object, my question is how can I get the provider from wagmi to use in the Web3js libraries Web3() object

**Comment by tmm** [comment]
wagmi does not support web3.js.
